app/javascript/components/DragAndDropOnField.jsx

```diff
@@ -0,0 +1,161 @@
+import React, { Component } from "react";
+import { Stage, Layer, Text, Group } from "react-konva";
+import PlayerChoice from "./PlayerChoice";
+
+export default class DragAndDropOnField extends Component {
+  setPlayers(playerNumber, team) {
+    var ourPlayerX = 100;
```

Comment:
`var` is old-school and there can be some problems with it. Use `const` when possible or `let` instead.